.github/workflows/publish-site.yml

```diff
@@ -0,0 +1,27 @@
+name: Website Build
+on:
+  push:
+    branches:
+      - 'feature/heavy-wip-website'
```

Comment:
Should probably add main here?

Reply:
Will do in a follow up PR immediately after this is merged.